packages/@aws-cdk/aws-efs/lib/efs-file-system.ts

```diff
@@ -171,6 +171,13 @@ export interface FileSystemProps {
    * @default RemovalPolicy.RETAIN
    */
   readonly removalPolicy?: RemovalPolicy;
+
+  /**
+   * Whether to enable automatic backups for the file system.
+   *
+   * @default - false
```

Comment:
```suggestion
   * @default false
```

we only start the string with a hyphen when there's a description to follow. If the default is a primitive value, you can just plug it in directly 🙂